### PR TITLE
[TS-492] Added 3 users in comparison

### PR DESCRIPTION
**Pull request by tungleu**
image

image

BONUS: 
changed webpage title and icon 

before: 
image
after: 
image

### CODE REVIEW

File: src/components/ComparisonTool/ComparisonModal.vue [hunk 1]
```diff
@@ -23,7 +23,7 @@
         data-cy="modal"
       >
         <v-card-title class="pl-0 pr-0">
-          Comparing Returns
+          Comparing Monthly Equity
```
Comment (itsShaVawn) [inline src/components/ComparisonTool/ComparisonModal.vue:26]:
```suggestion
          Comparing Equities - Last 4 Weeks
```

File: src/components/ComparisonTool/ComparisonModal.vue [hunk 2]
```diff
@@ -14,7 +14,7 @@
           medium
           v-on="on"
         >
-          COMPARE EQUITIES
+          COMPARE
```
Comment (itsShaVawn) [inline src/components/ComparisonTool/ComparisonModal.vue:17]:
```suggestion
          COMPARE WITH FAVORITES
```
maybe?

**Review by itsShaVawn — APPROVED** [review]

File: src/components/ComparisonTool/ComparisonModal.vue (hunk 2 above)
Comment (tungleu) [reply src/components/ComparisonTool/ComparisonModal.vue:17]: need consistency

**Review by rileykc98 — APPROVED** [review]
Beauty! Love the bonus content

**Pull request merged**